src/cpp/r/include/r/RSexp.hpp

```diff
@@ -125,8 +127,11 @@ void listNamedAttributes(SEXP obj, Protect *pProtect, std::vector<Variable>* pVa
 // weak/external pointers and finalizers
 bool isExternalPointer(SEXP object);
 bool isNullExternalPointer(SEXP object);
+SEXP makeTestExternalPointer(bool nullPtr);
```

Comment:
Maybe put this in a separate namespace (`test`?) or define it as a helper function in the test files where it's used, since this function isn't really appropriate for general development?

Reply:
I'll move it in a different namespace and remove at least the R function `.rs.makeExternalPointer` and define it where needed.